index.js

```diff
@@ -54,6 +54,8 @@ LogWatcher.prototype.start = function () {
   var fileSize = fs.statSync(this.options.logFile).size;
   var players = [];
   var gameOverCount = 0;
+  var data = '';
```

Comment:
Can you explain to me what this is used for? Why is it added to the buffer string and then set to the first item from the split result?